admin/class-aioseop-helper.php

```diff
@@ -0,0 +1,1128 @@
+		if ( current_user_can( 'aiosp_manage_seo' ) ) {
+			$this->_set_help_text( $module );
+		}
+	}
+
+	/**
+	 * Set this Help Text
+	 *
+	 * Sets the Help Text according to the module/class in use, but if there is
+	 * no class name in $module, then this Help Text will add all module help texts.
+	 *
+	 * @ignore
+	 * @since 2.4.2
+	 * @access private
+	 *
+	 * @param string $module All_in_One_SEO_Pack module.
+	 */
+	private function _set_help_text( $module ) {
+
+		switch ( $module ) {
+			case 'All_in_One_SEO_Pack':
+				$this->help_text = $this->help_text_general();
+				$this->help_text = array_merge( $this->help_text, $this->help_text_post_meta() );
+				break;
+			case 'All_in_One_SEO_Pack_Performance':
+				$this->help_text = $this->help_text_performance();
+				break;
+			case 'All_in_One_SEO_Pack_Sitemap':
+				$this->help_text = $this->help_text_sitemap();
+				break;
+			case 'All_in_One_SEO_Pack_Opengraph':
+				$this->help_text = $this->help_text_opengraph();
+				break;
+			case 'All_in_One_SEO_Pack_Robots':
+				$this->help_text = $this->help_text_robots_generator();
+				break;
+			case 'All_in_One_SEO_Pack_File_Editor':
+				$this->help_text = $this->help_text_file_editor();
+				break;
+			case 'All_in_One_SEO_Pack_Importer_Exporter':
+				$this->help_text = $this->help_text_importer_exporter();
+				break;
+			case 'All_in_One_SEO_Pack_Bad_Robots':
+				$this->help_text = $this->help_text_bad_robots();
+				break;
+			default:
+				$this->help_text = array_merge( $this->help_text, $this->help_text_general() );
+				$this->help_text = array_merge( $this->help_text, $this->help_text_performance() );
+				$this->help_text = array_merge( $this->help_text, $this->help_text_sitemap() );
+				$this->help_text = array_merge( $this->help_text, $this->help_text_opengraph() );
+				$this->help_text = array_merge( $this->help_text, $this->help_text_robots_generator() );
+				$this->help_text = array_merge( $this->help_text, $this->help_text_file_editor() );
+				$this->help_text = array_merge( $this->help_text, $this->help_text_importer_exporter() );
+				$this->help_text = array_merge( $this->help_text, $this->help_text_bad_robots() );
+				$this->help_text = array_merge( $this->help_text, $this->help_text_post_meta() );
+				break;
+		}
+
+		if ( AIOSEOPPRO ) {
+			$help_text = aioseop_add_pro_help( $this->help_text );
+
+			// TODO - Get Prefix from PRO Version.
+			foreach ( $help_text as $key => $text ) {
+				$this->help_text[ $key ] = $text;
+			}
+		}
+
+	}
+
+	/**
+	 * Help Text General Settings
+	 *
+	 * @ignore
+	 * @since 2.4.2
+	 * @access private
+	 *
+	 * @return array
+	 */
+	private function help_text_general() {
+		/*
+		 * Consider changing the construction of the macros.
+		 *
+		 * The name of the macro should NOT be inside _e() or __() because it does not make sense as it
+		 * won't change with the language.
+		 *
+		 * Moreover, it will confuse WPCS and it will try to replace %c (as in %category%) to %$1c.
```

Comment:
since this PR will be run through grunt (if it hasnt already), c will become %c. maybe you should add the codestandardsignore clause so that that does not happen

Reply:
Added. However, it would be best not to use `@codingStandardsIgnore`, and use `phpcs:disable <ruleset>`.

1. It allows us to identify which rules are being disabled.
2. It allows other rules to still be enabled.

Reply:
ignoring rules is required only for the block that has the sprint so it does not make sense to disable it for the entire file

Reply:
It is not disabled for the entire file. It is just for this code block.

Line 426 `//phpcs:enable`